src/main/kotlin/com/chattriggers/ctjs/minecraft/objects/gui/Gui.kt

```diff
@@ -156,6 +158,19 @@ abstract class Gui : GuiScreen() {
         onActionPerformed
     }
 
+    /**
+     * Registers a method to be run when the gui is opened.
+     * Arguments passed through to method:
+     * - the gui
+     *
+     * @param method the method to run
+     * @return the trigger
+     */
+    fun registerGuiOpened(method: Any) = run {
+        onGuiOpened = RegularTrigger(method, TriggerType.Other, getLoader())
+        onGuiOpened
+    }
```

Comment:
Let's remove the `Gui` from all of these identifiers, we're already in the Gui class so it's a bit redundant